pages/techgenius/tg-installation.md

```diff
@@ -62,7 +62,10 @@ If it does not work, try this:
 ```bash
     sudo apt-get install git
     sudo apt-get install virtualbox
-    sudo apt-get install vagrant
+    curl -o vagrant.deb https://releases.hashicorp.com/vagrant/2.0.1/vagrant_2.0.1_x86_64.deb?_ga=2.145355697.407736321.1516146560-799695961.1515792702
```

Comment:
I think that you should remove the ?_ga in this case, the links works without it anyway

Reply:
sounds good!